cmake/Modules/KatanaPythonSetupSubdirectory.cmake

```diff
@@ -227,20 +227,24 @@ function(add_python_setuptools_target TARGET_NAME)
   _generate_build_configuration_json(FILE_PREFIX ${TARGET_NAME}_ DEPENDS ${X_DEPENDS})
 
   # TODO(amp): The RPATH of the installed python modules will contain a reference to the build
-  #  directory. This is not ideal and could cause confusion, but without depending on an
-  #  additional command line tool there isn't any real way to fix it.
```

Comment:
To explain: the initial space makes my IDE view all the lines as a single Todo. Not important, but something you will find on most of my todos.